### ISSUE TITLE
Error 308 - A key component of SteamVR isn't working property.

### ISSUE THREAD

**Issue by FinitelyFailed** [ValveSoftware/SteamVR-for-Linux#132]
#### Your system information
* Steam client version (build number or date): Seo 8 2018, at 19:21:25
* Distribution (e.g. Ubuntu): Ubuntu 18.04
* Graphics driver version (run nvidia-settings): 396.54
* Gist for SteamVR System Information: https://gist.github.com/FinitelyFailed/fd6b66048950d1cf7fffef61be5a4083
* Opted into Steam client beta?: Yes
* Opted into SteamVR beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
When I start SteamVR I get:
"Hmm, that shouldn't have happend.
A key component of SteamVR isn't working properly.
(308)"

#### Steps for reproducing this issue:
1. Install SteamVR.
2. Start SteamVR.

**Comment by ryao** [comment]
I vaguely recall seeing something like this on Gentoo until I installed an undocumented dependency. I think it was the package that installed lsusb.

**Comment by FinitelyFailed** [comment]
I have that package (usbutils?) installed.

**Comment by FinitelyFailed** [comment]
I get this when I am just trying to start SteamVR, not a vr-game. :/

**Comment by FinitelyFailed** [comment]
I reinstalled Ubuntu (version 18.10 instead of 18.04) and I installed Steam och SteamVR. Everything worked fine. I than installed Unreal Engine 4.20 and that broke something and I got the same error as before. Now after a couple of updates to SteamVR later everything works fine again. I even installed Unreal Engine 4.21, and everything works better then ever. I never found out why SteamVR gave me that error ...
